app/javascript/shared/components/Radio.res

```diff
@@ -4,8 +4,8 @@ let str = React.string
 
 @react.component
 let make = (~id, ~label, ~onChange, ~checked=false) =>
-  <div className="relative">
-    <input className="w-0 h-0 absolute radio-input focus:outline-none" id type_="radio" onChange checked />
```

Comment:
@indicozy - This file has changes unrelated to this PR. Could you please stash or move them to a new PR?